lib/common.sh

```diff
@@ -222,7 +222,12 @@ export KUSTOMIZE_VERSION=${KUSTOMIZE_VERSION:-"v4.1.3"}
 
 # Kind version (if EPHEMERAL_CLUSTER=kind)
 export KIND_VERSION=${KIND_VERSION:-"v0.11.1"}
-export KIND_NODE_IMAGE_VERSION=${KIND_NODE_IMAGE_VERSION:-"v1.21.2"}
+
+if [ "${KUBERNETES_VERSION}" == "v1.21.2" ]; then
+  export KIND_NODE_IMAGE_VERSION="v1.21.2"
```

Comment:
This means, we will be pulling Kind node image (~400+ MB) every time we run into this condition. Because our host  image doesn't have  `v1.21.2` Kind node image and only `v1.22.0`.

Reply:
True. We shall keep both kind node image versions in the image since we'll be testing with both.

Reply:
Can you please make sure we inject 1.21.X Kind node image too in to the host image?

Reply:
https://github.com/Nordix/airship-dev-tools/pull/379